migrations/V26_update_dog_constraints.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE dogs
```

Comment:
The file name should have two underscores to delimit version from name. In this case—`V26__...`

Reply:
Ahh thanks!